Client/Frontend/Widgets/PhotonActionSheet/TrackingProtectionMenu.swift

```diff
@@ -139,37 +140,47 @@ extension PhotonActionSheetProtocol {
             label.font = DynamicFontHelper.defaultHelper.DeviceFontSmallBold
         }
         blockedtitle.customHeight = { _ in
-            return PhotonActionSheetUX.RowHeight - 15
+            return PhotonActionSheetUX.RowHeight - 10
         }
 
-        let xsitecookies = PhotonActionSheetItem(title: Strings.TPCrossSiteCookiesBlocked, iconString: "tp-cookie", accessory: .Disclosure) { action in
+        let xsitecookies = PhotonActionSheetItem(title: Strings.TPCrossSiteCookiesBlocked, iconString: "tp-cookie", accessory: .Disclosure) { action, _ in
             let title = String.localizedStringWithFormat(Strings.TPListTitle, action.title)
             let desc = Strings.TPCategoryDescriptionCrossSite
             self.showDomainTable(title: title, description: desc, blocker: blocker, categories: [BlocklistCategory.advertising, BlocklistCategory.analytics])
         }
-        let social = PhotonActionSheetItem(title: Strings.TPSocialBlocked, iconString: "tp-socialtracker", accessory: .Disclosure) { action in
+        let social = PhotonActionSheetItem(title: Strings.TPSocialBlocked, iconString: "tp-socialtracker", accessory: .Disclosure) { action,  _ in
             let title = String.localizedStringWithFormat(Strings.TPListTitle, action.title)
             let desc = Strings.TPCategoryDescriptionSocial
             self.showDomainTable(title: title, description: desc, blocker: blocker, categories: [BlocklistCategory.social])
         }
-        let fingerprinters = PhotonActionSheetItem(title: Strings.TPFingerprintersBlocked, iconString: "tp-fingerprinter", accessory: .Disclosure) { action in
+        let fingerprinters = PhotonActionSheetItem(title: Strings.TPFingerprintersBlocked, iconString: "tp-fingerprinter", accessory: .Disclosure) { action, _ in
             let title = String.localizedStringWithFormat(Strings.TPListTitle, action.title)
             let desc = Strings.TPCategoryDescriptionFingerprinters
             self.showDomainTable(title: title, description: desc, blocker: blocker, categories: [BlocklistCategory.fingerprinting])
         }
-        let cryptomining = PhotonActionSheetItem(title: Strings.TPCryptominersBlocked, iconString: "tp-cryptominer", accessory: .Disclosure) { action in
+        let cryptomining = PhotonActionSheetItem(title: Strings.TPCryptominersBlocked, iconString: "tp-cryptominer", accessory: .Disclosure) { action, _ in
             let title = String.localizedStringWithFormat(Strings.TPListTitle, action.title)
             let desc = Strings.TPCategoryDescriptionCryptominers
             self.showDomainTable(title: title, description: desc, blocker: blocker, categories: [BlocklistCategory.cryptomining])
         }
 
-        let addToWhitelist = PhotonActionSheetItem(title: Strings.TPDisableTitle, iconString: "menu-TrackingProtection-Off") { _ in
```

Comment:
@garvankeeley I'm not sure if this is the reason, but the identifier `menu-TrackingProtection-Off`to enable/disable TP is not working anymore (I run the tests on this branch on BB) it is found, but tapping on it does not enable/disable the switch. Could you please keep it? or add an ID for the switch button and make it visible (now I don't see it printed as an element 🙈 )

Reply:
Let me look at fixing this...